src/site/content/en/notifications/codelab-notifications-get-started/index.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: codelab
+title: Get started with the Notifications API
+description: |
+  In this codelab, learn how to request user permission and send notifications.
+date: 2019-10-15
+# Add an updated date to your post if you edit in the future.
+# updated: 2019-06-27
+glitch: codelab-notifications-get-started
+# Specify which file the glitch should start on.
+glitchPath: public/index.js
+related_post: use-push-notifications-to-engage-users
+---
+
+In this codelab, you'll use basic features of the
+[Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API) to:
+
+* Request permission to send notifications
+* Send notifications
+* Experiment with notification options 
+
+## Remix the sample app and view it in a new tab
+
+Notifications are automatically blocked from the embedded Glitch app, so you won't be able to preview the app on this page. Instead, here's what to do:
+
+{% Instruction 'remix', 'ol' %}
```

Comment:
Converted to our Instruction widget to make maintenance easier in future. (I removed the images since they're not usually provided.)